src/main/java/ai/timefold/solver/benchmarks/competitive/flowshop/FlowShopDataset.java

```diff
@@ -0,0 +1,661 @@
+    VFR800_20_10_Gap("vfr-large2015", "VFR800_20_10_Gap.txt", 800, 41574),
+    VFR800_20_1_Gap("vfr-large2015", "VFR800_20_1_Gap.txt", 800, 41558),
+    VFR800_20_2_Gap("vfr-large2015", "VFR800_20_2_Gap.txt", 800, 41407),
+    VFR800_20_3_Gap("vfr-large2015", "VFR800_20_3_Gap.txt", 800, 41425),
+    VFR800_20_4_Gap("vfr-large2015", "VFR800_20_4_Gap.txt", 800, 41426),
+    VFR800_20_5_Gap("vfr-large2015", "VFR800_20_5_Gap.txt", 800, 41710),
+    VFR800_20_6_Gap("vfr-large2015", "VFR800_20_6_Gap.txt", 800, 42010),
+    VFR800_20_7_Gap("vfr-large2015", "VFR800_20_7_Gap.txt", 800, 41425),
+    VFR800_20_8_Gap("vfr-large2015", "VFR800_20_8_Gap.txt", 800, 41492),
+    VFR800_20_9_Gap("vfr-large2015", "VFR800_20_9_Gap.txt", 800, 41796),
+    VFR800_40_10_Gap("vfr-large2015", "VFR800_40_10_Gap.txt", 800, 43861),
+    VFR800_40_1_Gap("vfr-large2015", "VFR800_40_1_Gap.txt", 800, 43671),
+    VFR800_40_2_Gap("vfr-large2015", "VFR800_40_2_Gap.txt", 800, 43746),
+    VFR800_40_3_Gap("vfr-large2015", "VFR800_40_3_Gap.txt", 800, 43749),
+    VFR800_40_4_Gap("vfr-large2015", "VFR800_40_4_Gap.txt", 800, 43892),
+    VFR800_40_5_Gap("vfr-large2015", "VFR800_40_5_Gap.txt", 800, 43905),
+    VFR800_40_6_Gap("vfr-large2015", "VFR800_40_6_Gap.txt", 800, 43811),
+    VFR800_40_7_Gap("vfr-large2015", "VFR800_40_7_Gap.txt", 800, 43766),
+    VFR800_40_8_Gap("vfr-large2015", "VFR800_40_8_Gap.txt", 800, 43839),
+    VFR800_40_9_Gap("vfr-large2015", "VFR800_40_9_Gap.txt", 800, 43879),
+    VFR800_60_10_Gap("vfr-large2015", "VFR800_60_10_Gap.txt", 800, 46244),
+    VFR800_60_1_Gap("vfr-large2015", "VFR800_60_1_Gap.txt", 800, 46470),
+    VFR800_60_2_Gap("vfr-large2015", "VFR800_60_2_Gap.txt", 800, 46493),
+    VFR800_60_3_Gap("vfr-large2015", "VFR800_60_3_Gap.txt", 800, 46389),
+    VFR800_60_4_Gap("vfr-large2015", "VFR800_60_4_Gap.txt", 800, 46457),
+    VFR800_60_5_Gap("vfr-large2015", "VFR800_60_5_Gap.txt", 800, 46401),
+    VFR800_60_6_Gap("vfr-large2015", "VFR800_60_6_Gap.txt", 800, 46421),
+    VFR800_60_7_Gap("vfr-large2015", "VFR800_60_7_Gap.txt", 800, 46319),
+    VFR800_60_8_Gap("vfr-large2015", "VFR800_60_8_Gap.txt", 800, 46474),
+    VFR800_60_9_Gap("vfr-large2015", "VFR800_60_9_Gap.txt", 800, 46538);
+
+    private final String module;
+    private final String filename;
+    private final BigDecimal bestSolution;
+    private final int jobs;
+
+    FlowShopDataset(String module, String filename, int jobs, int bestSolution) {
+        this.module = module;
+        this.filename = filename;
+        this.bestSolution = BigDecimal.valueOf(bestSolution);
+        this.jobs = jobs;
+    }
+
+    public int getJobs() {
+        return jobs;
+    }
+
+    @Override
+    public BigDecimal getBestKnownSolution() {
+        return bestSolution;
+    }
+
+    @Override
+    public boolean isLarge() {
+        return this.jobs > 100;
+    }
+
+    @Override
+    public boolean isBestKnownSolutionOptimal() {
+        return true;
```

Comment:
Is this true for every single dataset?

Reply:
In that case, the answer is "no". This should only return `true` when we know the result is actually optimal.